submissions/akulahere/html-css-popup/index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="../css/normalize.css.css" rel="stylesheet">
+    <link href="css/main.css" rel="stylesheet">
+    <title>Menu</title>
+</head>
+<body>
+    <nav>
+        <ul class = "navbar">
+            <li class = "nav-elem">
+                <a href="#">Mail</a>
+            </li>
+            <li class = "nav-elem">
+                <a href="#">Images</a>
+            </li>
+            <li class = "nav-elem">
+                <a href="#">
+                    <img src="img/popup-button.png">
+                </a>
+                <ul class="pop-up">
```

Comment:
In this case I recommend use `input` + `label` for better semantic and for usefulness.

Reply:
fixed

Reply:
Fixed with checkbox imput